include/engine/routing_algorithms/direct_shortest_path.hpp

```diff
@@ -19,30 +15,31 @@ namespace engine
 namespace routing_algorithms
 {
 
+template <typename AlgorithmT>
+InternalRouteResult
+directShortestPathSearch(SearchEngineData &,
+                         const datafacade::ContiguousInternalMemoryDataFacade<AlgorithmT> &,
+                         const std::vector<PhantomNodes> &)
+{
+    throw util::exception(std::string("directShortestPathSearch is not implemented for ") +
+                          typeid(AlgorithmT).name());
```

Comment:
Urgh I wouldn't use rtti - if you really need a string repr. I would simply add another trait to get a const char* literal